src/developers/quick_start.md

```diff
@@ -1,11 +1,13 @@
-This guide will take you through the steps of:
+# Quick Start
+
+In this guide we will set up a general-purpose TLS verifier, the `Notary`, so that a `Prover` can notarize some TLS data and generate a proof which he then shows to a `Verifier` for selective disclosure.
```

Comment:
```suggestion
In this guide we will set up a general-purpose TLS verifier ( a.k.a. the `Notary`), so that a `Prover` can notarize some TLS data and generate a proof which he then shows to a `Verifier` for selective disclosure.
```

Reply:
It had to pause to parse the meaning of the comma, I think parens convey the idea better.

Im beginning to think that a better name for the Notary is "TLS attestor" but we can keep "TLS verifier" for now.